src/MHTimer.js

```diff
@@ -501,6 +501,7 @@ function parseUserMessage(message) {
                 listRemind(message);
             else
                 addRemind(reminderRequest, message);
+            message.react('✅');
```

Comment:
Instead of a blanket response indicating success, we should reply with the checkmark only when the command was successful, and only when the response was sent to PM. When the bot sends a "failed to do the thing" PM, we should react to the source message with something that indicates failure, be it 😢, 🚫, ⛔, ❎, ✖️ or a MH discord custom emoji.

PS: ✔️ > ✅ 😉

Reply:
To build on this, it may make sense to have all commands that result in a message to a user return an object result containing the `message` object, whether a response was sent to dm (in which case a reaction on the message might be warranted), whether the outcome was a success, failure, or bot error, or even directly specify the emoji to react with. 

Then we'd wrap calls to these user invoked messages in a new method that receives that return signature and reacts to the original message appropriately.

So the function calls would end up looking like `provideFeedback(someFoo(args, for, foo));` instead of just `someFoo(args, for, foo);`

Reply:
Just added a new object (CommandData) to return from the functions that deal with commands that DM the user. Also put in a small if statement when those functions get called to react appropriately. Let me know what you think. Also also, I changed from the box check mark to the regular one

Reply:
Interestingly, on my phone the box check mark was a white version of the green check mark :D